CMakeLists.txt

```diff
@@ -159,6 +160,7 @@ SET(${PROJECT_NAME}_ALGORITHM_HEADERS
   algorithm/compute-all-terms.hpp
   )
 
+IF(${BUILD_PYTHON_INTERFACE} STREQUAL "ON")
```

Comment:
In fact I tried and the cmake failed as the value of  `IF(BUILD_PYTHON_INTERFACE)` is a sstring with value 'ON'
Maybe I missed something. Though it should work properly.